.github/workflows/test-cli.yml

```diff
@@ -0,0 +1,27 @@
+# Currently this does not contain any tests, only a license check
+# Once we add tests, we should add a step here to run them
+name: Run CLI Tests
+on: [push]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Setup Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: '1.19.6'
+      - name: Generate CLI
+        run: |
+          npm install
+          npm run generate.cli
+      - name: License check
+        uses: phrase/actions/lawa-ci
+        with:
+          github-token: ${{ secrets.GH_LAWA_ACCESS_TOKEN }}
+          decisions-file: .license-decisions.yml
+          ruby-version: 3.3.2
```

Comment:
Should we set this for non-Ruby projects? Maybe a default would be better here?

Reply:
you're right, no need to override this